excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreator.java

```diff
@@ -287,31 +290,31 @@ public String createErrorMessage(final CaseFieldEntityComplexACLValidationError
             error.getComplexFieldACLEntity(),
             def -> String.format(
                 INVALID_CRUD_VALUE_V_IN_TAB_T_CASE_TYPE_C_CASE_FIELD_F
-                    + "list element code '%s', user role '%s'. Detail: %s",
+                    + "list element code '%s', access profile '%s'. Detail: %s",
                 defaultString(def.getString(ColumnName.CRUD)),
                 def.getSheetName(),
                 def.getString(ColumnName.CASE_TYPE_ID),
                 def.getString(ColumnName.CASE_FIELD_ID),
                 def.getString(ColumnName.LIST_ELEMENT_CODE),
-                defaultString(def.getString(ColumnName.USER_ROLE)),
+                defaultString(def.getString(ColumnName.ACCESS_PROFILE)),
                 error.getDefaultMessage()));
     }
 
     @Override
     public String createErrorMessage(final CaseFieldEntityInvalidUserRoleValidationError error) {
         return newMessageIfDefinitionExists(error,
             error.getCaseFieldACLEntity(),
-            def -> CaseRoleServiceImpl.isCaseRole(defaultString(def.getString(ColumnName.USER_ROLE)))
+            def -> CaseRoleServiceImpl.isCaseRole(defaultString(def.getString(ColumnName.ACCESS_PROFILE)))
                 ? String.format(
                 "Invalid case role '%s' in %s tab, case type '%s', case field '%s'. "
                     + DEFINED_IN_CASE_ROLES_SHEET_VERIFICATION_MESSAGE,
-                defaultString(def.getString(ColumnName.USER_ROLE)),
+                defaultString(def.getString(ColumnName.ACCESS_PROFILE)),
                 def.getSheetName(),
                 def.getString(ColumnName.CASE_TYPE_ID),
                 def.getString(ColumnName.CASE_FIELD_ID)) :
                 String.format(
-                    "Invalid idam role '%s' in %s tab, case type '%s', case field '%s', crud '%s'",
-                    defaultString(def.getString(ColumnName.USER_ROLE)),
+                    "Invalid IDAM access profile '%s' in %s tab, case type '%s', case field '%s', crud '%s'",
```

Comment:
Remove the word IDAM

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f